src/helpers/numberFormat.ts

```diff
@@ -1,6 +1,5 @@
-export function numberFormatter(language: string | undefined, humanize: boolean, round: boolean) {
-  return new Intl.NumberFormat(language, {
-    ...(humanize ? { notation: 'compact', compactDisplay: 'short' } : {}),
-    ...(round ? { maximumFractionDigits: 0 } : {}),
-  }).format
+import { numeral } from '../i18n'
+
+export function numberFormatter(humanize: boolean, round: boolean) {
+  return (value: number) => numeral(value).format(`0${!round ? '.[000]' : ''}${humanize ? 'a' : ''}`)
```

Comment:
Hi Rich, For non humanize to get 1000000 -> 1,000,000, I think a prefix `0,` is required. i.e. `format(`0,0${!roun`...
![Screen Shot 2020-03-26 at 4 23 04 PM](https://user-images.githubusercontent.com/6854428/77706011-718f4a80-6f7e-11ea-96c2-4c8e7058160d.png)

Reply:
@gj262 @rcbevans Strangely, this library gives small "m" for millions in English and German, but capital "M" for French. I don't get it. Looks like "meters" to me.